src/test/java/uk/ac/ebi/eva/commons/models/metadata/DatasetTest.java

```diff
@@ -1,7 +1,13 @@
-package embl.ebi.variation.commons.models.metadata;
+package uk.ac.ebi.eva.commons.models.metadata;
 
 import org.junit.Test;
 
+import uk.ac.ebi.eva.commons.models.metadata.Analysis;
+import uk.ac.ebi.eva.commons.models.metadata.Array;
+import uk.ac.ebi.eva.commons.models.metadata.Dataset;
+import uk.ac.ebi.eva.commons.models.metadata.FileGenerator;
+import uk.ac.ebi.eva.commons.models.metadata.Run;
```

Comment:
I think all these imports shouldn't be necessary, since they are in the same package. The same applies to all the metadata test classes.